src/components/HistoryTable/HistoryTable.stories.tsx

```diff
@@ -13,7 +16,12 @@ export default {
   decorators: [
     (Story: () => JSX.Element) => (
       <MockedProvider>
-        <Story />
+        <VirtuosoMockContext.Provider
+          // eslint-disable-next-line react/jsx-no-constructed-context-values
+          value={{ viewportHeight: 500, itemHeight: COMMIT_HEIGHT }}
+        >
+          <Story />
+        </VirtuosoMockContext.Provider>
       </MockedProvider>
```

Comment:
The TaskHistoryTable and VariantHistoryTable stories aren't working for me, should they be updated or removed?

Reply:
Fixed